src/commands/createNewProject/CSharpProjectCreator.ts

```diff
@@ -25,6 +25,8 @@ export class CSharpProjectCreator extends ProjectCreatorBase {
     private _debugSubpath: string;
     private _runtime: ProjectRuntime;
 
+    private _detectedRuntime: boolean = false;
```

Comment:
nit: _detectedRuntime sounds like it'd hold the runtime to me.  What about _hasDetectedRuntime so we know it's a state?